admin_manual/configuration_database/linux_database_configuration.rst

```diff
@@ -208,7 +208,7 @@ like this:
   extension=pdo_pgsql.so
   extension=pgsql.so
 
-  [PostgresSQL]
+  [PostgreSQL]
```

Comment:
Cool the docs example is even broken, `[PostgreSQL]` is correct :see_no_evil:

Reply:
Yep. 

Codespell changed `PostgresSQL` to `Postgressql` and I then checked the value and corrected it.

Reply:
Should backport this line at least. Or you want to backport codespell completely?